usaspending_api/api_docs/markdown/endpoints.md

```diff
@@ -42,6 +42,8 @@ To reduce unnecessary data transfer, most endpoints return a default set of info
 | [/api/v1/accounts/awards/](/api/v1/accounts/awards/) | GET, POST | Returns a list of financial account data by treasury account symbol, program activity, object class, and award |
 | /api/v1/accounts/awards/:id | GET, POST | Returns a single treasury account symbol/program activity/object class/award record with all fields |
 | [/api/v1/accounts/awards/total/](/api/v1/accounts/awards/total/) | POST | Supports aggregation on treasury account symbol/program activity/object class/award records |
+| [/api/v1/award_spending/award_type/](/api/v1/award_spending/award_type/) | GET | Returns a list of total obligations by award_type |
```

Comment:
I'm confused - this doc lists them as in `v1` but below they're going in as `v2`?